etc/workflows/attach-watson-transcripts.xml

```diff
@@ -33,12 +33,24 @@
       exception-handler-workflow="partial-error"
       description="Distribute and publish to engage server">
       <configurations>
+        <configuration key="download-source-flavors">dublincore/*,security/*</configuration>
```

Comment:
Why are you updating dublincore and security catalogs here? The `attach-watson-transcription` operation should not touch any metadata.